ERROR - 2024-04-30 23:55:32 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert(Array)
#4 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#5 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-04-30 23:55:32 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombe`, `apellido`, `email`, `usuario`, `pass`) VALUES (:nombe:, :apellido:, :email:, :usuario:, :pass:)', [...], false)
 3 APPPATH\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
